packages/composite-checkout/src/components/checkout-payment-methods.tsx

```diff
@@ -91,9 +91,15 @@ export default function CheckoutPaymentMethods( {
 	}
 	debug( 'rendering paymentMethods', paymentMethods );
 
+	const isCardOnly = paymentMethods.length === 1 && paymentMethods[ 0 ].id === 'card';
```

Comment:
This is tricky because it presupposes that there will always be a payment method with the ID `'card'`, but the documentation for this package has no such requirements. The `@automattic/composite-checkout` package is agnostic about where it's used and in fact is currently also used for our payment method management system. It may soon also power checkout in wp-admin. We shouldn't put any wpcom checkout specific code in here.

Is there a way we can do this without relying on a specific ID? Maybe we could pass `className` into this component from calypso checkout instead?